temporal/t.shift/t.shift.py

```diff
@@ -69,7 +69,7 @@ def main():
     stds = tgis.open_old_stds(name, type, dbif)
     check = stds.shift(gran=gran, dbif=dbif)
 
-    if check == False:
+    if check is False:
```

Comment:
Here the `is operator` is used for comparison (`check is False`), while in almost all other cases `== False` is simply removed (`not check`)

Reply:
Good catch. That's intentional here, because I wanted to be make less assumptions about the values coming from the "check" function and in the `check` variable. When the function name and variable name are something like `is_ok()` and `ok`, then it is pretty clear the possible values are True and False. With check, as a reader of this code, it is not clear if `check == False` and `not check` are really the same, because check could be e.g. None if the check failed and the case should be handled in some other way. (I don't know if that's the case here, but I also don't need to know right now.) With `is_ok()` and `ok` it is safer to make the assumption about what the possible values are. Both `not check` and `check is False` are allowed, so my decision here was using whatever is safer and easier to review.